getinsomnia/insomnia
# feat: enable RequestAuth in pre-request scripting [INS-3379]

Changes:
- [x] enabled RequestAuth in pre-request scripting


### How to test:

Please run this script and check its output.
```
const { RequestAuth } = require('insomnia-collection');

const reqAuth = new RequestAuth({
		type: 'basic',
		basic: [
				{ key: 'username', value: 'user1' },
				{ key: 'password', value: 'pwd1' },
		],
});

const result = reqAuth.toJSON();
console.log(result);
```
